app/views/ideas/show.html.erb

```diff
@@ -61,12 +61,12 @@
 <div id="votes" class="column-one-third">
   <% if @idea.approved_by_admin? %>
     <p class="govuk-body-l">Votes</p>
-    <p class="govuk-body-l"><%= @idea.votes.count %></p>
+    <p id="vcount" class="govuk-body-l"><%= @idea.votes.count %></p>
     <div id="votebutton">
       <% if @idea.user_voted?(current_user) %>
-        <%= button_to 'Remove Vote', idea_vote_path(@idea, @vote.id), method: :delete, class: "govuk-button" %>
+        <%= button_to 'Remove Vote', idea_vote_path(@idea, @vote.id), method: :delete, class: "govuk-button", id: "removevote" %>
```

Comment:
Can we have a consistent naming system for HTML ids?
Taking a look at the repo I see both "an-id" and "an_id" being used.

We've taken a look at the Apply code, and they're using "-"s to separate words.  We'd vote for that. 👍